src/app/(proper_react)/(redesign)/(public)/breaches/page.tsx

```diff
@@ -30,6 +30,9 @@ export function generateMetadata() {
 
 export default async function Page() {
   const allBreaches = await getBreaches();
+  const breachesNewestFirst = allBreaches.sort((a, b) => {
+    return +new Date(b.BreachDate) - +new Date(a.BreachDate);
```

Comment:
Could you add a comment what these pluses do here? It's not directly obvious to me. Is that a way to implicitly convert it to a number? If so, could we be more explicit, so people won't have to know what the type cast happens to do?

```suggestion
    return (new Date(b.BreachDate)).getTime() - (new Date(a.BreachDate)).getTime();
```

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4757/commits/46032bc138fa154892c610fc9e7fdc6c5ea99247